tests/manage/mcg/test_mcg_log_level.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+
+from logging import getLogger
+
+from ocs_ci.framework import config
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import (
+    get_noobaa_core_pod,
+    get_noobaa_endpoint_pod,
+    get_noobaa_operator_pod,
+    get_pod_logs,
+)
+from ocs_ci.framework.testlib import tier2, bugzilla
+from ocs_ci.ocs.constants import (
+    CONFIGMAP,
+    NOOBAA_CONFIGMAP,
+    POD,
+    NOOBAA_APP_LABEL,
+    STATUS_RUNNING,
+)
+
+log = getLogger(__name__)
+NAMESPACE = config.ENV_DATA.get("cluster_namespace", "openshift-storage")
+
+
+def get_noobaa_cfg_log_level(cfgmap: OCP) -> str:
+    """
+    Gets noobaa current log level from its configmap
+
+    Args:
+        cfgmap (OCP): OCP object of kind "configmap"
+
+    Returns:
+        str: value of NOOBAA_LOG_LEVEL
+
+    """
+    return cfgmap.get(NOOBAA_CONFIGMAP).get("data").get("NOOBAA_LOG_LEVEL")
+
+
+def set_noobaa_cfg_log_level(cfgmap: OCP, log_level: str) -> None:
+    """
+    Patch noobaa configmap, to set log level
+
+    Args:
+        cfgmap (OCP): OCP object of kind "configmap"
+        log_level (str): value of NOOBAA_LOG_LEVEL to set
+
+    """
+    cmd = f'{{"data": {{"NOOBAA_LOG_LEVEL": "{log_level}"}}}}'
+    cfgmap.patch(resource_name=NOOBAA_CONFIGMAP, params=cmd, format_type="merge")
```

Comment:
Please log the change for clarity:

```suggestion
    log.info(f"Setting noobaa's log level to {log_level} at {NOOBAA_CONFIGMAP}:")
    cmd = f'{{"data": {{"NOOBAA_LOG_LEVEL": "{log_level}"}}}}'
    cfgmap.patch(resource_name=NOOBAA_CONFIGMAP, params=cmd, format_type="merge")
```

Reply:
fixed